open_lm/params.py

```diff
@@ -742,6 +742,17 @@ def parse_args(args):
         action="store_true",
         help="If set, allow model to do multiple data passes over our dataset, in order to reach the desired number of tokens.",
     )
+    parser.add_argument(
+        "--use-smp-flash-attention",
+        type=int,
+        default=None,
+        help="Using SMP Flash Attention.",
+    )
+    parser.add_argument(
+        "--sharding-strategy",
```

Comment:
is this used? if so can we have a more specific name?

Reply:
also not seeing where `--use-smp-flash-attention` is used

Reply:
This is not used for FP8. Just placeholder flags defaulted to `None` for Sagemaker Model Parallel.

Reply:
Removed this to avoid confusion